tests/test_find_mapservice.py

```diff
@@ -0,0 +1,26 @@
+from openspoor import FeatureServerOverview
+import pytest
+
+
+@pytest.fixture(scope='session')
+def all_featureserver_layers():
+    return FeatureServerOverview()
+
+
+# TODO: These tests can be a lot stricter
+def test_get_all_featureserver_layers(all_featureserver_layers):
```

Comment:
Deze test faalt als mapservices een nieuwe feature toevoegd, terwijl dat deze functionaliteit niet zou breken. Misschien kan je deze test aanpassen dat die checkt dat bepaalde features die vaak gebruikt worden voorkomen in de lijst .

Reply:
De tests zijn nu wat minder streng geworden; zo lang er nu enige layers gevonden worden (en het aantal layers met het woord spoor erin er meer zijn dan met het woord hek erin) dan komen de tests er doorheen. Voor de rest durfde ik het niet aan dat het aantal layers toe zal blijven nemen in de toekomst. 
Op het moment dat er geen featureservers meer bestaan werkt heel openspoor niet meer, en aangezien er nu meer dan 10 keer zoveel toepassingen zijn met spoor dan met hek durf ik die check voorlopig ook nog wel aan (het is nu 41 om 3).